scss/styles.scss

```diff
@@ -1,3 +1,15 @@
 * {
```

Comment:
check indentation in this whole file. 
Remember that indentation should be `2 spaces or 1 tab`